deploy/database/schema.game.sql

```diff
@@ -84,10 +84,10 @@ CREATE TABLE game_chat_log (
     game_id            MEDIUMINT UNSIGNED NOT NULL,
     chat_time          TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     chatting_player    SMALLINT UNSIGNED NOT NULL,
-    -- A chat message is limited to 500 Unicode characters (see
+    -- A chat message is limited to 2000 Unicode characters (see
     -- GAME_CHAT_MAX_LENGTH in BMInterface and Game.js). Since a Unicode
```

Comment:
If GAME_CHAT_MAX_LENGTH was actually moved from BMInterface to ApiSpec, which it seems like based on what you changed in this pull, this comment should be updated to say so.